doc/sphinx/source/env_variables.rst

```diff
@@ -5,6 +5,10 @@ The behavior of pocl can be controlled with multiple environment variables
 listed below. The variables are helpful both when using and when developing
 pocl.
 
+- **POCL_AFFINITY**
+ If set to 1, each thread of the pthread CPU driver sets its affinity to
```

Comment:
What are the benefits of this setting? Can you add some more documentation so people know when and why this should be used.

Reply:
performance-wise probably none, but it allows to easily verify that splitting pthread into subdevices works properly (and it's just a few lines).